docs/content/docs/test/Toast.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Testing toasts
+type: docs
+---
+
+# Testing toasts
+
+[android.widget.Toast](https://developer.android.com/reference/android/widget/Toast)
+
+{{< hint warning>}}
+Testing toasts is a hard because other toasts can show up and yours will be queued. 
+We hack app with proxy object to mitigate this.
+{{< /hint >}}
+
+{{< hint warning>}}
+It is also recommended to use [snackbar](https://developer.android.com/reference/com/google/android/material/snackbar/Snackbar), 
+or your custom way to display brief unintrusive messages instead. So you should consider not testing it at all.
+{{< /hint >}}
+
+## Proxy toast
+
+Use provided extension methods to be able to spy on `showToast` functions.
+
+```kotlin
+import com.avito.android.util.showToast
+
+showToast("I'am a toast!")
+```
+
+```kotlin
+implementation("com.avito.android:proxy-toast:$version")
+```
+
+## ToastRule
+
+```kotlin
+@get:Rule
+val toastRule = ToastRule()
+
+fun test() {
+    toastRule.checks.toastDisplayedWithText("I'am a toast!")
+}
+```
+
+```kotlin
+androidTestImplementation("com.avito.android:toast-rule:$version")
+```
```

Comment:
It's look like abstract bull shit.
Point of this page isn't clear.
It's not clear that `showToast` must be used in production instead of standart functions.
It's not clear that Rule useless without `showToast`
Info about SnackBar is too large compare with other content.
Structure i see:
- You can't test toast
- We help you to test toast
- You can test toast that shows only by our `showToast` in production
- You can use our rule
- We recommend to use snackbar

Reply:
> It's look like abstract bull shit.

Please, do not use such judgments. The problem is, it looks aggressive and doesn't provide information about exact problems.
https://avito-tech.github.io/avito-android/docs/contributing/codereview/#best-practices

Apart from this, I agree with the next proposals.

Reply:
Sorry @dsvoronin . I want to say that text is hard to read. I will work on my messages

Reply:
Do i need to create pr with proposals?